### PR TITLE
Azure DevOps documentation

### PR DESCRIPTION
**Pull request by andrebriggs**

### CODE REVIEW

**Comment by andrebriggs** [comment]
Added documentation and merging

File: gitops/azure-devops/AzureDevOpsGit.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+**CAVEAT**: Since we own the destination repo there are easier ways to do auth here! Let's see how our shell script has changed to support ADO
+
+Afterward hit "Save & Queue"
+You will see the build run and hopefully complete successfully
+At this point we can make a PR change to the HLD repo
+
+
+
+## Outline of instructions (ignore)
```
Comment (samiyaakhtar) [inline gitops/azure-devops/AzureDevOpsGit.md]: Is this intentional?

Comment (andrebriggs) [reply gitops/azure-devops/AzureDevOpsGit.md]: Yes for the moment. This merge to to unblock @yradsmikham

Comment (samiyaakhtar) [reply gitops/azure-devops/AzureDevOpsGit.md]: Gotcha, I assume you're pulling in changes from master into services, correct?

**Pull request merged**